pkg/arm/data/master-startup.sh

```diff
@@ -567,20 +591,56 @@ spec:
     securityContext:
       privileged: true
     volumeMounts:
-    - mountPath: /etc/origin/master
+    - mountPath: /etc/origin/master/
       name: master-config
-      readOnly: true
-    - mountPath: /etc/origin/cloudprovider
+    - mountPath: /etc/origin/cloudprovider/
       name: master-cloud-provider
-      readOnly: true
+    - mountPath: /etc/containers/registries.d/
+      name: signature-import
+    - mountPath: /usr/libexec/kubernetes/kubelet-plugins
+      mountPropagation: HostToContainer
+      name: kubelet-plugins
   hostNetwork: true
+  priorityClassName: system-node-critical
   volumes:
   - hostPath:
-      path: /etc/origin/master
+      path: /etc/origin/master/
     name: master-config
   - hostPath:
       path: /etc/origin/cloudprovider
     name: master-cloud-provider
+  - hostPath:
+      path: /etc/containers/registries.d
+    name: signature-import
+  - hostPath:
+      path: /usr/libexec/kubernetes/kubelet-plugins
+    name: kubelet-plugins
+EOF
+
+cat >/etc/origin/master/recycler_pod.yaml <<'EOF'
```

Comment:
Why is this placed here vs the addons bindata?

Reply:
because its need to be available on API startup time. Just to increase startup time.

Reply:
Does the api server depend on it?

Reply:
https://github.com/openshift/openshift-azure/pull/664/files#diff-12aca7ad0dbbd92fddf4bc4d0d4842bdR210 yes and no. It can be opted out If we dont use storage functionality.

Reply:
Now, I see. Thanks